data_handler/handlers/liquidable_debt/debt_handlers.py

```diff
@@ -120,267 +72,100 @@ def get_prices_range(collateral_token_name: str, current_price: Decimal) -> Iter
+        )
+        state.debt_interest_rate_models = InterestRateModels(
+            init_value=Decimal(interest_rate_models.debt.get("USDC", ""))
+        )
+
         current_prices = Prices()
         asyncio.run(current_prices.get_lp_token_prices())
-        INTEREST_RATE_MODELS_MAPPING = {
-            protocol.value.lower(): self.get_interest_rate_models_data(protocol_name=protocol.value)
-            for protocol in AvailableProtocolID
-        }
 
-        # init just once for improving performance
-        state = self.state_class()
-        # TODO: 1. Read data from db for state.loan_entities from LoanState model
-        # TODO: 2. Set last interest rate record by block
-        # state.interest_rate_models = InterestRateModels(..)
-        # # Set up collateral and debt interest rate models
-        # state.collateral_interest_rate_models = InterestRateModels(
-        #     values=interest_rate_models[COLLATERAL_FIELD_NAME.lower()]
-        # )
-        # state.debt_interest_rate_models = InterestRateModels(
-        #     values=interest_rate_models[DEBT_FIELD_NAME.lower()]
-        # )
-        # for index, row in data.items():
-            # even if it isn't per-user data, we need to provide a user ID
-            # so like that we're able to provide debt and collateral values
-            # TODO use just one collateral token and debt token
-            # for collateral_token in row[COLLATERAL_FIELD_NAME].keys():
-            #     hypothetical_collateral_token_prices = self.get_prices_range(
-            #         collateral_token_name=collateral_token,
-            #         current_price=current_prices.prices.values[collateral_token]
-            #     )
+        hypothetical_collateral_token_prices = self.get_prices_range(
+            collateral_token_name="STRK",
+            current_price=current_prices.prices.values["STRK"]
+        )
 
+        result_data = dict()
         # Go through first hypothetical prices and then through the debts
         for hypothetical_price in hypothetical_collateral_token_prices:
-            current_prices.prices.values[collateral_token] = hypothetical_price
-
-            if not isinstance(state, (HashstackV0State, HashstackV1State)):
-                # FIXME not needed, will be done in the first step
-                # state.loan_entities[user_wallet_id].debt.values = {
-                #     debt_token: hypothetical_price
-                #     for debt_token in TOKEN_PAIRS.get(collateral_token, "")
-                # }
-                # state.loan_entities[user_wallet_id].collateral.values = {
-                #     collateral_token: hypothetical_price
-                # }
-
             liquidable_debt = state.compute_liquidable_debt_at_price(
-                prices=TokenValues(init_value=current_prices.prices.values.get(collateral_token)),
-                collateral_token=collateral_token,
+                prices=TokenValues(values=current_prices.prices.values),
```

Comment:
```suggestion
                prices=current_prices.prices,
```
This doesn't work?

Reply:
No, because prices should be a `TokenValues` instance